### ISSUE TITLE
Use standard identifiers for references

### ISSUE THREAD

**Issue by cthoyt** [MobleyLab/GuthrieSolv#4]
Typically, people expect DOIs, PubMed identifiers (PMIDs), or arXiv identifiers for publications. The bibliography in this repository is a bit old school, which means that the provenance information associated with each row of the database isn't really machine-readable nor actionable. It would be a great contribution if someone could untangle all of the references in https://github.com/MobleyLab/GuthrieSolv/blob/master/guthrie_references_and_status.csv and replace them with DOIs or PMIDs.

**Comment by davidlmobley** [comment]
Yes, totally agree. I wonder if there's a way we can use the crossref API to at least attempt to pull some of these automatically.

I also happened across this tool, which is oriented towards fixing bib files, but relates... https://github.com/nschloe/betterbib
